config/initializers/templates.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.config.to_prepare do
+  if ActiveRecord::Base.connection.table_exists?("goldencobra_templates")
+    # Beim starten der Application wird das Verzeichnis /app/views/layouts durchsucht nach
+    # neuen Templatefiles und erzeugt für diese dann eine Goldencobra::Template
+    Goldencobra::Template.templates_for_select.each do |template_file_name|
+      template = Goldencobra::Template.where(layout_file_name: template_file_name).first
+      unless template
+        template = Goldencobra::Template.create(layout_file_name: template_file_name, title: template_file_name)
+        Goldencobra::Articletype.all.each do |at|
+          at.templates << template
```

Comment:
Warum ist at.save hier nicht erforderlich?

Reply:
`articletype has_many :templates` wird über die << Operator um ein weiteres Element ergänzt. 
Es kommt also nur ein weiterer Eintrag in die Kreuztabelle `articletype_teplates` hinzu. Das Object `at` vom Typ `articletype` wird dabei nicht verändert und muss daher auch nicht neu gespeichert werden